authors/apps/authentication/tests/test_login.py

```diff
@@ -11,11 +12,17 @@
 class LoginTest(BaseTest):
     def setUp(self):
         super().setUp()
-        self.register_test_user()
+        self.resp = self.register_test_user()
 
     def test_login_successful(self):
```

Comment:
I'll add on to what @malep2007 noted and mention separation of concerns. This test in particular seems to combine testing of two independent functionality; `activation` and `login`.